### PR TITLE
Remove 4.4.0 release notes

### PR DESCRIPTION
**Pull request by joragua**
## Related Issues
App:

- [x] Add changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
- [x] Add feature to Release Notes in `ReleaseNotesViewModel.kt` creating a new `ReleaseNote()` with String resources (if required)

_____

### CODE REVIEW

File: owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt
Comment (JuancaG05) [inline owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt:1]: Apart from removing them from here, you should remove them from the `strings.xml` file

**Review by JuancaG05 — CHANGES_REQUESTED** [review]

File: owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt
Comment (joragua) [reply owncloudApp/src/main/java/com/owncloud/android/presentation/releasenotes/ReleaseNotesViewModel.kt:1]: Done 😄

File: owncloudApp/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -726,20 +726,6 @@
...
     <string name="release_notes_icon">Release note icon</string>
 
-    <string name="release_notes_4_3_0_title_accessibility_improvements">Improvements in accessibility</string>
-    <string name="release_notes_4_3_0_subtitle_accessibility_improvements">Some improvements to make the application more accessible</string>
-    <string name="release_notes_4_4_0_title_improved_from_original_folder_auto_upload">Auto uploads more consistent</string>
-    <string name="release_notes_4_4_0_subtitle_improved_from_original_folder_auto_upload">Added some mechanisms over retries to improve reliability</string>
-    <string name="release_notes_4_4_0_title_resharing_capability">Improvements in private shares</string>
-    <string name="release_notes_4_4_0_subtitle_resharing_capability">The \"can share\" option is now only shown when editing a private share if the user is able to edit it</string>
-    <string name="release_notes_bugfixes_title">Minor bugfixes</string>
-    <string name="release_notes_bugfixes_subtitle">Some bugs were fixed to improve experience in the app</string>
```
Comment (JuancaG05) [inline owncloudApp/src/main/res/values/strings.xml:736]: Let's not remove these 2 strings since they are generic and useful for every release 😃

Comment (joragua) [reply owncloudApp/src/main/res/values/strings.xml:736]: Added 👍

**Review by JuancaG05 — APPROVED** [review]
LGTM 👍

**Comment by jesmrec** [comment]
LGTM too 👍 

No QA required here, beyond the green CI

**Pull request merged**